### DISCUSSION TITLE
Alpha overlay seems not worked properly

### DISCUSSION THREAD

**Discussion by nfurtin** [visualdesigncafe/nature-shaders#12] [Help]
I tried to fade out blocked camera view trees by using nature overlay with alpha 0.5. But result looking like alpha 0 or 1. Any way to get half transparent material? 
![Снимок экрана 2024-05-02 233744](https://github.com/visualdesigncafe/nature-shaders/assets/168734954/42c7d967-e43f-4ecc-8f94-039a1311c930)
![Снимок экрана 2024-05-02 233823](https://github.com/visualdesigncafe/nature-shaders/assets/168734954/977a779c-af95-4072-911f-515f46065eef)
![Снимок экрана 2024-05-02 233905](https://github.com/visualdesigncafe/nature-shaders/assets/168734954/efae04c4-0a4f-455b-ab52-5a280c63ca58)
![Снимок экрана 2024-05-02 234022](https://github.com/visualdesigncafe/nature-shaders/assets/168734954/2dfe5462-dee1-498a-b92d-f54727a0b387)

**Comment by maxplooi** [comment]
The alpha works as a threshold, so everything above the threshold is visible and everything below is hidden. What you can do is edit the alpha channel of the bark texture and add some noise to it. Right now the alpha channel of the bark is a solid color, so either the entire bark is above the threshold or below. With some noise in it, it will fade out more gradually.
